README.md

```diff
@@ -1,54 +1,30 @@
-[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
-# BHoM
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) [![Build status](https://ci.appveyor.com/api/projects/status/woeecxaxa3vbeu6f/branch/master?svg=true)](https://ci.appveyor.com/api/projects/status/mongo_toolkit/branch/master) [![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/Mongo_Toolkit/Mongo_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=107&branchName=master)
```

Comment:
Azure badge should be `[![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/Mongo_Toolkit/Mongo_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=105&branchName=master)`